app/assets/javascripts/discourse/components/user-card-contents.js.es6

```diff
@@ -140,7 +140,12 @@ export default Component.extend(CardContentsBase, CanCheckEmails, CleansUp, {
     this._positionCard($target);
     this.setProperties({ visible: true, loading: true });
 
-    const args = { stats: false };
+    const args = {};
+
+    if (this.siteSettings.enable_new_user_card_route) {
```

Comment:
Minor but in Javascript it's generally a good idea for performance to keep all objects the same shape. It would be preferred to do `args.forCard = this.siteSettings.enable_new_user_card_route`. Use `false` over `undefined`.